stubs/Modules/Base/App/Filament/Resources/RedirectResource.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+namespace App\Filament\Resources;
+
+use App\Enums\RedirectType;
+use App\Filament\Resources\RedirectResource\Pages;
+use App\Filament\Resources\RedirectResource\RelationManagers;
+use App\Models\Redirect;
+use Filament\Forms\Components\Select;
+use Filament\Forms\Components\Textarea;
+use Filament\Forms\Components\TextInput;
+use Filament\Forms\Form;
+use Filament\Resources\Resource;
+use Filament\Tables;
+use Filament\Tables\Table;
+
+class RedirectResource extends Resource
+{
+    protected static ?string $model = Redirect::class;
+
+    protected static ?string $navigationIcon = 'heroicon-o-rectangle-stack';
+
+    public static function getNavigationLabel(): string
+    {
+        return __('Redirects');
+    }
+
+    public static function getLabel(): ?string
+    {
+        return __('Redirect');
+    }
+
+    public static function getPluralLabel(): ?string
+    {
+        return __('Redirects');
+    }
+
+    public static function form(Form $form): Form
+    {
+        return $form
+            ->columns(1)
+            ->schema([
+                TextInput::make('request_path')
+                    ->label(__('Request path'))
+                    ->startsWith('/')
+                    ->required(),
+
+                TextInput::make('target_path')
+                    ->label(__('Target path'))
+                    ->startsWith(['/', 'http', 'https'])
+                    ->required(),
```

Comment:
Het is misschien wel tof om hier een custom validatie in te doen die checkt bestaat er al een redirect met omgekeerde target en request path? Zodat we daarmee direct valideren dat we geen infinite loops kunnen aanmaken

Reply:
De infinite loops die eykenburg had aangemaakt was omdat ze van `/pagina` naar `/pagina` redirect hadden aangemaakt... deze case kan ik nog wel even makkelijk afvangen

Reply:
Prima, dat is inderdaad ook iets dat moet worden afgevangen, maar dat veranderd mijn initiele vraag niet.

Het zou nog steeds mooi zijn als je hier via de validatie voorkomt dat er een infinite loop kan ontstaan

Reply:
Even naar gekeken, lijkt niet makkelijk te doen.
Heb een custom rule aangemaakt en probeer daarin de tegenhanger op te halen, alleen is die niet makkelijk te achterhalen. Normaal zou ik hiervoor `request()->get()` gebruiken, maar die leeg. Vermoedelijk omdat het livewire gebruikt?